guild.go

```diff
@@ -1191,6 +1190,21 @@ func (g guildQueryBuilder) GetRoles() ([]*Role, error) {
 	return getRoles(r.Execute)
 }
 
+// Search through GetRoles to find a role
+func (g guildRoleQueryBuilder) Get() (*Role, error) {
+	guild := g.client.Guild(g.gid)
+	roles, err := guild.GetRoles()
+	if err != nil {
+		return &Role{}, err
+	}
+	for _, role := range roles {
+		if role.ID == g.roleID {
+			return role, nil
+		}
+	}
+	return &Role{}, nil
```

Comment:
This should return an error saying "role not found" or something.

Reply:
done!